optimade/server/models/modified_jsonapi.py

```diff
@@ -0,0 +1,26 @@
+from optimade.server.models.jsonapi import Link, Links, Source
+
+from datetime import datetime
+from pydantic import  UrlStr, BaseModel
+from typing import Optional, Union, Any
+
+class Attributes(BaseModel):
+    local_id: UrlStr
+    lad_modified: datetime
+    immutable_id: Optional[UrlStr]
+
+class ErrorLinks(BaseModel):
+    about: Union[Link, UrlStr]
+
+class Error(BaseModel):
+    id: Optional[str]
+    status: Optional[str]
+    code: Optional[str]
+    title: Optional[str]
+    detail: Optional[str]
+    source: Optional[Source]
+    meta: Optional[dict]
+    links: Optional[ErrorLinks]
+
+class Links(Links):
```

Comment:
I find this *very* ugly, redefinition of the same name when importing something? Looks like a problem waiting to happen

Reply:
you could simply `from .jsonapi import Links as JSONLinks`

Reply:
My preference here would be to just import the namespace and then use `jsonapi.Links` as the base class.

Reply:
My preference here would be to just import the namespace and then use `jsonapi.Links` as the base class.

Reply:
even better!